### ISSUE TITLE
RS Requester Over-requesting with Occultism storage

### ISSUE THREAD

**Issue by Rodg88** [EnigmaticaModpacks/Enigmatica6#4114]
### Modpack Version

0.5.25 dev

### Describe your issue.

With this setup to test, and a pattern to make 9 sand to 9 glass:

![image](https://user-images.githubusercontent.com/2611674/150680404-08199df6-a66e-495a-b670-372becc6dba8.png)

If I request to stock less than a stack, it does so without issue.

![image](https://user-images.githubusercontent.com/2611674/150680442-66f1523a-5e80-47ff-aa66-d7f07a52afe4.png)

Then I set it to keep 500 in stock, and it does that correctly. However once it's filled the 500 order, it requests another 250
![image](https://user-images.githubusercontent.com/2611674/150680530-2aea69ab-26b7-48aa-8a04-de380f5e4180.png)
Then after that another 125, and then another stack, and then a half stack/quarter stack. Then it requests ~440 after that and continues the cycle, requesting half the last request again.

Up to 2.6k and counting:
![image](https://user-images.githubusercontent.com/2611674/150680725-cff1dbb9-b228-4c4b-b656-d20b7eb7ab22.png)

I set the requester to ask for 65, and it's continually requesting one craft at a time after getting the first 64.

It works fine with a drawer setup instead:

![image](https://user-images.githubusercontent.com/2611674/150680155-abc9c7fa-c3c9-4cba-a1cf-714f6d798fbe.png)

So basically I guess it's looking for a number of stacks + extra, but the stack size of 1024 in occultism storage is messing up the calculation.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by NillerMedDild** [comment]
@Darkere halp pls! Who do I ask nicely to fix this? 😄 Occultism? The Buuz?

**Comment by Darkere** [comment]
@Buuz135
